stride/physics/boundaries/devito.py

```diff
@@ -5,6 +5,43 @@
 from .boundary import Boundary
 
 
+class SpongeBoundaryElastic(Boundary):
```

Comment:
name of the class should be 'SpongeBoundary1' for consistency

Reply:
Correction made